dev-libs/ferrisloki/files/ferrisloki-3.0.13-r3-configure-libsigc.patch

```diff
@@ -0,0 +1,66 @@
+Rollup of src_prepare() from previous revisions plus additional fixes
+
+ * derice damn configure script
+ * fix building with libsgc++-2.6
+ * fix building with /bin/sh -> dash
```

Comment:
Please add bug mention (for example `bug #888888`) for each change (for those you can).

Reply:
done